### ISSUE TITLE
Accessibility: Tabbing not working in Firefox

### ISSUE THREAD

**Issue by jancavan** [Automattic/wp-calypso#23102]
There are no visible `:focus` styles when tabbing around Calypso using FF. It looks like the styles are actually there, but they just aren't visible; not even on the master or any sidebar.

FF v58.0.2, MacOS High Sierra v10.13.3

Related: https://github.com/Automattic/wp-calypso/issues/21889

/cc @MichaelArestad

**Comment by sixhours** [comment]
Was looking into this, and it appears to be because of a MacOS setting:

<img width="667" alt="screen shot 2018-03-30 at 2 00 50 pm" src="https://user-images.githubusercontent.com/2124984/38148161-45f2eeec-3423-11e8-916d-7841de04047e.png">

Switching this to "All controls" allows focus styles to appear for all types of elements (links, etc) in Firefox. Chrome does its own thing, FF is using these system settings.

**Comment by jancavan** [comment]
Thanks @sixhours! That's interesting to know. For other sites like Facebook, Twitter, etc., it seems to be working fine even without "All controls" enabled, so I wonder if they do something to get this to work (JS maybe?).

**Comment by sixhours** [comment]
Possibly, though I struggled to tab through Facebook and Twitter in Firefox until I turned on that OS setting. There are no focus styles on links without it. I'm using High Sierra with FF 59 if that helps.

Any areas that do show focus styles when that setting is off are button elements rather than anchors. I was able to test this in Calypso by swapping the Write anchor for a button.

**Comment by youbrokesomething** [comment]
Firefox now has much better support for focus styles; this issue should be resolved.
